src/app/frontend/logs/stateconfig.js

```diff
@@ -27,16 +27,10 @@ import {stateName} from './state';
  */
 export default function stateConfig($stateProvider) {
   $stateProvider.state(stateName, {
-    url: `${appendDetailParamsToUrl('/log')}/:container`,
-    params: {
-      'container': {
```

Comment:
No. It has changed.

https://ui-router.github.io/guide/ng1/migrate-to-1_0#non-optional-path-parameters